common/app/common/cms/CmsService.java

```diff
@@ -0,0 +1,19 @@
+package common.cms;
+
+import play.libs.F;
+
+import java.util.Locale;
+
+/**
+ * Service that provides page content, usually coming from a Content Management System (CMS).
+ */
+public interface CmsService {
+
+    /**
+     * Gets the page content corresponding to the given key for a certain language.
+     * @param locale for the localized text
+     * @param pageKey identifying the page
+     * @return the promise of the page content identified by the key, in the given language
+     */
+    F.Promise<CmsPage> get(final Locale locale, final String pageKey);
```

Comment:
This maybe not be enough. For example sometimes you need to pass some flags like: ```isPreview``` or any other helper parameter.